core-services/prow/02_config/_plugins.yaml

```diff
@@ -80,6 +80,14 @@ approve:
   repos:
   - openshift-knative
   require_self_approval: false
+- ignore_review_state: false
+  repos:
+  - openshift-psap/special-resource-operator
+  require_self_approval: true
+- lgtm_acts_as_approve: true
+  repos:
+  - openshift-psap/special-resource-operator
+  require_self_approval: false
```

Comment:
Two separate stanzas for a signle repo (with conflicting `require_self_approval` clauses) does not seem right.

Reply:
I took the example from the other repos in this same file.

Reply:
No other repo seems to be present in two different stanzas (and if it is, then it's a bug).